content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -319,4 +319,46 @@ There are at least two workarounds:
 ```bash
 kubectl taint nodes NODE_NAME role.kubernetes.io/master:NoSchedule-
 ```
+
+## When the Linux distribution for a Node has a read-only mounted /usr directory
```

Comment:
nit: This heading stands out from the others by starting with “When”.
Consider either future-proofing:
```suggestion
## When the Linux distribution for a Node has a read-only mounted /usr directory {#usr-mounted-read-only}
```

or also update the title text, eg:
```suggestion
## `/usr` is mounted read-only on nodes {#usr-mounted-read-only}
```